sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_users_first_seen_v1/query.sql

```diff
@@ -1,20 +1,20 @@
 WITH _current AS (
-  SELECT
+  SELECT DISTINCT
     user_id,
-    ARRAY_AGG(DISTINCT service IGNORE NULLS) AS services_used,
+    -- TOOD: should this should be dropped? We have fxa_users_services_first_seen to answer this kind of question.
+    -- ARRAY_AGG(DISTINCT service IGNORE NULLS) AS services_used,ß
```

Comment:
The way to answer this is three parts:
1. Are people using this field? If so, keep it.

2. Is it possible queries that would target this field can be answered with this new table + `fxa_users_services_first_seen`. For example, asking for the first seen date for users of a specific service:
```
SELECT ...
FROM fxa_user_first_seen_v1
JOIN fxa_users_services_first_seen USING (user_id)
WHERE fxa_users_services_first_seen.service = '$service'
```

3. Does this field add an incremental ease of querying for common query patterns over (2.)? If so, consider keeping it.

Reply:
I think my main concern is that logically I do not think `services_used` belongs in this model and should live in some kind of `fxa_users_profile` or `fxa_users_usage_overview` model instead. To me first_seen implies we record the state as it at the creation time and is immutable.

Also, this model to me can be a bit counter-intuitive. In that we have a date with a corresponding list of services that user has used in general and not on that date specifically. Unless the end user of this model has that knowledge the model could be easily misinterpreted. 

I hope that explained the rationale behind proposing this change.

Reply:
I'll leave it to you then :) Misuse b/c of misunderstanding is a very good thing at avoid. Just be cautious about changing the schema in case people are using that field.

Reply:
Will do, and I do think this field as a quick access field could have value. So we still should provide it, but I think it should live elsewhere. I'll make the rest of the changes to have this PR ready and will only merge once I'm confident it won't break anything.

Reply:
So the only place where I can see that field (`services_used`) being referenced is `fxa_amplitude_export_v1`. Do you happen to know @fbertsch if we still need this table? I did not find any usage when I did a global search for it inside bqetl, looker-spoke and looker-hub.

Also, tried an org wide search on Github: https://github.com/search?q=org%3Amozilla%20fxa_amplitude_export_v1&type=code

In Looker it seems to be a hidden field:
https://github.com/search?q=org%3Amozilla+services_used&type=code

My impression is it only existed to support `fxa_amplitude_export_v1`, is there anything you think I missed?

Reply:
We _don't_ need the amplitude export anymore, since we don't export to Amplitude. If `services_user` isn't otherwise used, then we can probably drop it.